scripts/core/store_learning.py

```
@@ -216,13 +220,47 @@ async def store_learning_v2(
+                    )
+                    # result is "UPDATE N" — check if row was actually updated
+                    try:
+                        parts = (result or "").split()
+                        if len(parts) == 2 and parts[0] == "UPDATE" and int(parts[1]) > 0:
+                            superseded_id = supersedes
+                    except (ValueError, IndexError):
+                        pass
+            except Exception:
+                # Graceful: don't fail the store if chain update fails
+                # (e.g., superseded_by column doesn't exist yet)
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)